pol-core/bscript/compiler/astbuilder/UserFunctionVisitor.cpp

```diff
@@ -1,20 +1,21 @@
 #include "UserFunctionVisitor.h"
 
+#include "bscript/compiler/ast/ClassDeclaration.h"
 #include "bscript/compiler/ast/UserFunction.h"
 #include "bscript/compiler/astbuilder/BuilderWorkspace.h"
 #include "bscript/compiler/model/CompilerWorkspace.h"
 
 namespace Pol::Bscript::Compiler
 {
 UserFunctionVisitor::UserFunctionVisitor( const SourceFileIdentifier& sfi, BuilderWorkspace& ws )
-  : workspace( ws ), tree_builder( sfi, ws )
+    : workspace( ws ), tree_builder( sfi, ws )
 {
 }
 
 antlrcpp::Any UserFunctionVisitor::visitFunctionDeclaration(
     EscriptGrammar::EscriptParser::FunctionDeclarationContext* ctx )
 {
-  auto uf = tree_builder.function_declaration( ctx );
+  auto uf = tree_builder.function_declaration( ctx, "" );
```

Comment:
add comment: visitFunctionDeclaration does not get triggered for class declarations, so the class name is always empty.